test/serializers/development_serializer_test.rb

```diff
@@ -1,20 +1,30 @@
 require 'test_helper'
 
 class DevelopmentSerializerTest < ActiveSupport::TestCase
+  def development
+    @_d ||= Development.create!(
+      id: 101010, name: 'Gadfly Hotel', address: '505 Washington Street', city: 'Boston', state: 'MA', zip_code: '02111', status: 'in_construction', commsf: 12, estemp: 75, private: true, created_at: Time.new('1969-12-31 19:00:00 -0500'), updated_at: Time.new('1969-12-31 19:00:00 -0500'), year_compl: 2016, creator: users(:normal)
```

Comment:
Line is too long. [331/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.